docs/guide/siteaccess.md

```diff
@@ -32,7 +32,7 @@ Typical uses of a SiteAccess are:
 The back-office UI of eZ Platform is housed in a predefined `admin` SiteAccess in `admin_group`.
 
 If you have a multisite setup with a separate back-office interface for each site,
-you need to create your own admin SiteAccesses for each site individually and add them to this group.
+you need to create your own admin SiteAccesses and add them to this group. In cases where the sites are on separate databases they'll need own repository config _(incl own storage and search connection)_, var dir, cache pool, as well as separate varnish/fastly cache in case that is in use for each site individually.
```

Comment:
`for each site individually` can be somewhat misunderstood, afaik host header is taken into account for http cache so it only needs to be separated per repo, or where you thinking of some issue here?

Reply:
actually var dir and varnish/fastly could be optional and configured in system.default for all siteaccesses (or groups). But in this context mandatory setting are repository config, DBs and cache pools. I could add an optionally settings note for storage and varinsh/fastly.